calc/token.py

```diff
@@ -6,9 +6,16 @@
 # Saved by the test: When these constants were defined the token strings were
 # inadvertently switched. The ``test_eof_token_at_end_of_line`` test caught the
 # error.
-INTEGER, EOF, PLUS = 'INTEGER', 'EOF', 'PLUS'
+INTEGER, EOF = 'INTEGER', 'EOF'
+PLUS, MINUS, TIMES, DIVIDED_BY = "PLUS", "MINUS", "TIMES", "DIVIDED_BY"
 
 
+# Rabbit Hole: Pylint is a program that takes in as input a Python file. It'll
```

Comment:
Nah, a rabbit hole is a rabbit hole.  Students can ask for clarification but other wise I plan on not discussing them.